builtin/logical/pki/crl_util.go

```diff
@@ -196,7 +196,7 @@ func (cb *crlBuilder) checkForAutoRebuild(sc *storageContext) error {
 		// error.
 		defaultPeriod, defaultErr := time.ParseDuration(defaultCrlConfig.AutoRebuildGracePeriod)
 		if defaultErr != nil {
-			return fmt.Errorf("error checking for auto-rebuild status: unable to parse duration from both config's grace period (%v) and default grace period (%v):\n- config: %v\n- default: %v\n", cfg.AutoRebuildGracePeriod, defaultCrlConfig.AutoRebuildGracePeriod, err, defaultErr)
+			return fmt.Errorf("error checking for auto-rebuild status: unable to parse duration from both config's grace period (%v) and default grace period (%v):\n- config: %v\n- default: %w\n", cfg.AutoRebuildGracePeriod, defaultCrlConfig.AutoRebuildGracePeriod, err, defaultErr)
```

Comment:
I think there should be two %w here not one?  Both of those looks like errors to me.

Reply:
Apparently you can only wrap one error with `fmt.Errorf`, not two. So I chose the last one to wrap in this place and one other (`pki/backend.go` -- auto-tidy and crl stuff in the `periodicFunc`).

Should both just be `%v` instead?

Reply:
*sigh*, no I think that's correct.  I'm sad at go here, not you (and absolutely didn't realize that).

Reply:
Cool, I'll merge then. Thank you!